typings/index.d.ts

```diff
@@ -9,6 +9,7 @@ export function renderHook<P, R>(
   readonly result: {
     current: R
   }
+  readonly nextUpdate: () => Promise<void>
```

Comment:
not sure if `Promise<void>` is the most correct type here?

Reply:
Looks ok to me

Reply:
If we pass the value of the hook's result to nextUpdate/nextValue. I think that the signature should be `readonly nextValue: () => Promise<R>`